source/_includes/common-tasks/data_disk.md

```diff
@@ -64,3 +64,41 @@ The system will prepare the data disk and immediately reboot. The reboot will ta
 Using an USB attached SSD can draw quite some power. For instance on Raspberry Pi 3 the official Raspberry Pi power supply (PSU) only provides 2.5A which can be too tight. Use a more powerful power supply if you experience issues. Alternatively use a powered USB hub. Connect the Hub to one of the USB slots of your Raspberry Pi, and connect the SSD to the Hub. In this setup the power supply of the Hub will power the attached device(s).
 
 </div>
+
+### Migrating an external data disk to another system
+
+This section shows how to move an external data disk from one system to another.
+This can be an option if the following elements apply to your use case:
+
+- You already have a functioning Home Assistant instance (system 1) that is using an external data disk.
+- You have another, new, Home Assistant instance (system 2).
+- You want now want to use the data disk of system 1 on system 2 instead.
+
+The aim is to migrate the data from system 1 to system 2. One way to do this is by [restoring a backup](/common-tasks/os/#restoring-a-backup). The other way is to move the data disk. This can be an interesting option if you have a large amount of data on your external disk or if your external disk has more storage capacity than your new system.
+
+#### Prerequisites
+
+- A Home Assistant instance using an external data disk (system 1)
+- A Home Assistant instance to which you want to move the external data disk (system 2)
+
+#### To migrate an external data disk to another system
+
+To migrate an external data disk from one system to another, follow these steps:
+
+1. [Create a backup](/common-tasks/os/#backups) of both systems and store these backups on another system.
```

Comment:
We advertise the feature as a quick way to migrate, but then still ask for a backup :sweat_smile: 

It does make sense, but maybe we can weaken it a bit, e.g.

```suggestion
1. [Create a backup](/common-tasks/os/#backups) of both systems and store these backups on another system (not strictly necessary, but recommended just in case, at least for the important data).
```